lab-steven/app/config/router-config.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = [
+  '$stateProvider',
+  '$urlRouterProvider',
+  function($stateProvider, $urlRouterProvider) {
+    $urlRouterProvider.when('', '/join#signup');
```

Comment:
I know we reviewed this after this lab, but a reminder that `$urlServiceProvider.rules.when()` is the newer, acceptable format for these redirects.